Ginger/Ginger/SolutionWindows/TargetApplicationsPage.xaml.cs

```diff
@@ -266,9 +287,30 @@ private void btnClearAll_Click(object sender, RoutedEventArgs e)
         {
             foreach (ApplicationPlatform applicationPlatform in WorkSpace.Instance.Solution.ApplicationPlatforms.ToList())
             {
-                if (WorkSpace.Instance.SolutionRepository.GetAllRepositoryItems<BusinessFlow>().Any(x => x.TargetApplications.Any(y => y.Name == applicationPlatform.AppName)))
+                bool doesAppExistInBF = WorkSpace.Instance.SolutionRepository.GetAllRepositoryItems<BusinessFlow>().Any(x => x.TargetApplications.Any(y => y.Name == applicationPlatform.AppName));
```

Comment:
this code is repetitive, please create a function out of it and use it wherever it is needed